x-pack/plugins/security_solution/public/timelines/components/flyout/pane/pane.styles.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { css } from '@emotion/css';
+import { Global } from '@emotion/react';
+import { useEuiTheme, euiAnimFadeIn, transparentize, euiBackgroundColor } from '@elastic/eui';
+
+export const usePaneStyles = () => {
+  const EuiTheme = useEuiTheme();
+  const { euiTheme } = EuiTheme;
+  return css`
+    // euiOverlayMask styles
+    position: fixed;
+    top: var(--euiFixedHeadersOffset, 0);
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 10vh;
+    animation: ${euiAnimFadeIn} ${euiTheme.animation.fast} ease-in;
+    background: ${transparentize(euiTheme.colors.ink, 0.5)};
+    z-index: ${euiTheme.levels.flyout};
+
+    &.timeline-wrapper--hidden {
+      display: none;
+    }
+
+    .timeline-flyout {
```

Comment:
We should change this name to better match the modal, but you don't have to worry about it in this PR

Reply:
I haven't renamed timeline-related "flyout" words because there are a lot of occurrences, and honestly, I didn't know what other name to use. 
Timeline is a bit of everything, it behaves like a modal in some way, but it is not a modal ([guidelines](https://eui.elastic.co/#/layout/modal/guidelines)), and it is built with flyout parts on the inside, but it is not a flyout either.
Maybe we should use a generic name, like `timeline-container`, `–wrapper`, `-pane`, `-layer`... in all those places, but that's probably a team decision.

Reply:
Yea, that's fine. We'll deal with it in a refactor/tech debt ticket at some point